src/base/memory_monitor_sr.cpp

```diff
@@ -674,7 +672,7 @@ void mmon_aggregate_server_info (MMON_SERVER_INFO &info)
 void mmon_aggregate_module_info (int module_index, std::vector<MMON_MODULE_INFO> &info)
 {
   assert (mmon_Gl != nullptr);
-  assert (module_index < (int)MMON_MODULE_LAST && module_index >= 0);
+  assert (module_index < (int)MMON_MODULE_LAST && module_index >= -1);
```

Comment:
Rather than using `-1` directly, how about using some meaningful constant value ? 
ex) MMON_MODULE_ALL ?

Reply:
I believe that if there is a place where that meaning should be included, it would be `MMON_MODULE_ID`. However, I am uncertain whether placing that information here would be appropriate, so I opted to leave it as a raw value.

Reply:
I think that it's not bad to put `MMON_MODULE_ALL` into the enum. At your choice.

Reply:
I add it.